### PR TITLE
test(elasticity): add mini elasticity test for pre-testing

### PR DESCRIPTION
**Pull request by soyacz**
Sometimes when working on improvement in performance tests it's needed to test it.

Added mini elasticity test configuration to allow such testing and is fast and cheap.

### Testing

- [X] - tested as part of https://github.com/scylladb/scylla-cluster-tests/pull/8733

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: configurations/performance/cql-stress-650gb-8-col-i4i-mini-test-throughput-oss.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+prepare_write_cmd: [ "cql-stress-cassandra-stress write no-warmup cl=ALL n=6500000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=500 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..6500000",
```
Comment (vponomaryov) [inline configurations/performance/cql-stress-650gb-8-col-i4i-mini-test-throughput-oss.yaml:1]: How this configuration is tied to the `OSS`  part in the file name?

Comment (soyacz) [reply configurations/performance/cql-stress-650gb-8-col-i4i-mini-test-throughput-oss.yaml:1]: it's just a copy of oss file (as ent and oss differ in throttling).
Maybe for this test it doesn't matter as it's not official, but why caring?

Comment (fruch) [reply configurations/performance/cql-stress-650gb-8-col-i4i-mini-test-throughput-oss.yaml:1]: just leave a comment inside of it, to point which configuration it's based on.

you can drop `oss` in the process, it doesn't make any difference

Comment (soyacz) [reply configurations/performance/cql-stress-650gb-8-col-i4i-mini-test-throughput-oss.yaml]: fixed

**Review by vponomaryov — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**